src/charts/CytoViz/components/DetailsLog/DetailsLog.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import useStyles from './style';
+
+function prettyPercentage(val) {
+  return (val * 100).toFixed(1) + '%';
+}
+
+function hiddenElements(cytoElements) {
+  let hiddenElements = 0;
+  cytoElements.forEach((element) => {
+    if (element.data('hidden')) {
+      hiddenElements++;
+    }
+  });
+  return hiddenElements;
+}
+
+const DetailsLog = (props) => {
+  const { cytoAsState } = props;
+  const classes = useStyles();
+
+  const compounds = cytoAsState.nodes().parent().length;
+  const children = cytoAsState.nodes().nonorphans().length;
+  const noCompounds = cytoAsState
+    .nodes()
+    .nonorphans()
+    .union(cytoAsState.nodes().orphans())
+    .subtract(cytoAsState.nodes().parent());
+  const edges = cytoAsState.edges().length;
+  const hiddenCompounds = hiddenElements(cytoAsState.nodes().parent());
+  const hiddenNodes = hiddenElements(noCompounds);
+  const hiddenEdges = hiddenElements(cytoAsState.edges());
```

Comment:
It seems we're using several times `cytoAsState.nodes().nonorphans()` and `cytoAsState.nodes().parent()`, maybe we could create intermediate variables to store them